Assets/__Scripts/MapEditor/Input/BeatmapNoteInputController.cs

```diff
@@ -34,6 +34,7 @@ public class BeatmapNoteInputController : BeatmapInputController<BeatmapNoteCont
     };
 
     [SerializeField] private NoteAppearanceSO noteAppearanceSO;
+    public static bool QuickModificationActive;
```

Comment:
does this need to be static? It would persist when the scene is closed; if you hold alt while exiting, and edit a new song, CM will think you are still quick editing until you press alt again